armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -156,6 +156,7 @@ def performThermalAxialExpansion(
         tempGrid: list,
         tempField: list,
         setFuel: bool = True,
+        coldHeightsToHot: bool = False,
```

Comment:
Should add `coldHeightsToHot` to the docstring

Reply:
@john-science it'd be nice if our linting would catch this. What do you think?

Reply:
https://github.com/terrapower/armi/pull/1342/commits/73c16ed9d0a14ffbf3090e619426e6c740fc38b4